File: z3. balanced-parathesis-problem/parenthesis.js
var data = '((blah)))'
	;

var balanced_parenthesis = function(str){
	var count=0;

	str = str.split('');

	str.map(function(char){
		if(char=='(') count+=1;
		if(char==')') count-=1;
	});

	if(count===0){
		return 'balanced!';
	} else {
		return 'unbalanced :(';
	}

};

console.log(balanced_parenthesis(data));

File: z13. balanced-parathesis-problem/parenthesis.py
data = '(blah))'

def balanced_parenthesis(str):
	count=0
	if(len(str) == 0):
		print('need to pass in a string')
		return

	for char in str:
		if char == '(':
			count+=1
		if char == ')':
			count-=1

	if(count != 0):
		return 'unbalanced!'
	else:
		return 'balanced!'




if __name__ == '__main__':
	print(balanced_parenthesis(data))
